plugins/Installation/lang/en.json

```diff
@@ -19,145 +19,145 @@
-        "SystemCheckPackHelp": "The pack() function is required to track visitors in Matomo.",
-        "SystemCheckParseIniFileHelp": "This built-in function has been disabled on your host. Matomo will attempt to emulate this function but may encounter further security restrictions. Tracker performance will also be impacted.",
-        "SystemCheckPdoAndMysqliHelp": "On a GNU\/Linux server you can compile php with the following options: %1$s In your php.ini, add the following lines: %2$s",
+        "SystemCheckPageSpeedDisabled": "PageSpeed is turned off",
+        "SystemCheckPageSpeedWarning": "It is recommended to turn off the PageSpeed module on your %s web server: PageSpeed causes several issues with Matomo, such as: broken pages reports, broken row evolution, etc. Please turn off mod_pagespeed on this server.",
+        "SystemCheckPackHelp": "The 'pack()' function is required to track visitors in Matomo.",
+        "SystemCheckParseIniFileHelp": "This built-in function has been turned off on your host. Matomo will attempt to emulate it, but may encounter further security restrictions. Tracker performance will be impacted.",
+        "SystemCheckPdoAndMysqliHelp": "On a GNU\/Linux server you can compile PHP with the following options: %1$s In your php.ini, add the following lines: %2$s",
         "SystemCheckPhp": "PHP version",
-        "SystemCheckPhpPdoAndMysqli": "More information on: %1$sPHP PDO%2$s and %3$sMYSQLI%4$s.",
-        "SystemCheckSessionHelp": "You need to configure and rebuild PHP with \"session\" support enabled (don't use --disable-session).",
+        "SystemCheckPhpPdoAndMysqli": "More info: %1$sPHP PDO%2$s and %3$sMYSQLI%4$s.",
+        "SystemCheckSessionHelp": "Configure and rebuild PHP with the required \"session\" support (don't use --disable-session).",
         "SystemCheckSettings": "Required PHP configuration (php.ini)",
         "SystemCheckSummaryNoProblems": "Huzzah! There are no problems with your Matomo setup. Give yourself a pat on the back.",
         "SystemCheckSummaryThereWereErrors": "Uh-oh! Matomo has detected some %1$scritical issues%2$s with your Matomo setup. %3$sThese issues should be fixed immediately.%4$s",
-        "SystemCheckSummaryThereWereWarnings": "There are some issues with your system. Matomo will run, but you might experience some minor problems.",
-        "SystemCheckTimeLimitHelp": "On a high traffic website, executing the archiving process may require more time than currently allowed. If necessary, change the max_execution_time directive in your php.ini file.",
+        "SystemCheckSummaryThereWereWarnings": "There are some issues with your system. Matomo will run, but you might experience minor problems.",
+        "SystemCheckTimeLimitHelp": "On a high-traffic website, executing the archival process may take longer than currently allowed. If necessary, change the 'max_execution_time' directive in your php.ini file.",
```

Comment:
```suggestion
        "SystemCheckTimeLimitHelp": "On a high traffic website, executing the archiving process may take longer than currently allowed. If necessary, change the 'max_execution_time' directive in your php.ini file.",
```

Reply:
The website isn't high, whereas the traffic is.